lib/arclight/custom_document.rb

```diff
@@ -10,6 +10,8 @@ class CustomDocument < SolrEad::Document
       t.unitid(path: 'archdesc/did/unitid', index_as: %i[displayable])
       t.repository(path: 'archdesc/did/repository/corpname/text() | archdesc/did/repository/name/text()', index_as: %i[displayable facetable])
       t.creator(path: "archdesc/did/origination[@label='creator']/*/text()", index_as: %i[displayable facetable])
+      t.function(path: 'archdesc/controlaccess/function/text()', index_as: %i[displayable facetable])
+      t.occupation(path: 'archdesc/controlaccess/occupation/text()', index_as: %i[displayable facetable])
```

Comment:
Are `function` or `occupation` solr fields that we're generating here being used?

I see that we're also parsing them directly via xpath in the `access_subjects` method below.  Sorry if this is just my OM/Solrizer naivety on how these things work.